Resources/config/services.yml

```diff
@@ -121,3 +121,11 @@ services:
       - [setSerializer, ['@jms_serializer']]
       - [setMessageFactory, ['@smartesb.message_factory']]
     public: '%kernel.debug%'
+
+  # PHPAMQPLIB
+  smartesb.drivers.queue.phpamqplib:
+    class: Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\PhpAmqpLibDriver
+    calls:
+      - [setSerializer, ['@jms_serializer']]
+      - [setMessageFactory, ['@smartesb.message_factory']]
+    public: '%kernel.debug%'
```

Comment:
Any reason to set it public based on the kernel env? In the AMQP one this is done to get it from the container directly when testing it.

Reply:
No reason. I was just following the implementations for the old driver